wren-ui/src/components/pages/home/promptThread/AnswerResult.tsx

```diff
@@ -201,95 +227,101 @@ export default function AnswerResult(props: Props) {
-              </Text>
-            </>
-          }
-        >
-          <ChartAnswer
-            threadResponse={threadResponse}
-            onRegenerateChartAnswer={onGenerateChartAnswer}
-            onAdjustChartAnswer={onAdjustChartAnswer}
-          />
-        </Tabs.TabPane>
-      </StyledTabs>
-      <div className="mt-2">
-        <Tooltip
-          overlayInnerStyle={{ width: 'max-content' }}
-          placement="topLeft"
-          title={knowledgeTooltip}
-        >
-          <Button
-            type="link"
+    <div style={resultStyle} data-jsid="answerResult">
+      <QuestionTitle className="mb-4" question={question} />
+      <Preparation
+        className="mb-3"
+        {...preparation}
+        data={threadResponse}
+        isAnswerPrepared={isAnswerPrepared}
+      />
+      {showAnswerTabs && (
+        <>
+          <StyledTabs
+            className="select-none"
+            type="card"
             size="small"
-            className="mr-2"
-            onClick={() =>
-              onOpenSaveToKnowledgeModal(
-                { question, sql },
-                { isCreateMode: true },
-              )
-            }
-            data-guideid="save-to-knowledge"
+            onTabClick={onTabClick}
           >
-            <div className="d-flex align-center">
-              <RobotSVG className="mr-2" />
-              Save to Knowledge
-            </div>
-          </Button>
-        </Tooltip>
-        <ViewBlock
-          view={view}
-          onClick={() => onOpenSaveAsViewModal({ sql, responseId: id })}
-        />
-      </div>
-      {renderRecommendedQuestions(
-        isLastThreadResponse,
-        recommendedQuestionProps,
-        recommendedQuestionsProps.onSelect,
+            {!isBreakdownOnly && (
+              <Tabs.TabPane
+                key={ANSWER_TAB_KEYS.ANSWER}
+                tab={
+                  <>
+                    <CheckCircleFilled className="mr-2" />
+                    <Text>Answer</Text>
+                  </>
+                }
+              >
+                <TextBasedAnswer {...props} />
+              </Tabs.TabPane>
+            )}
+            <Tabs.TabPane
+              key={ANSWER_TAB_KEYS.VIEW_SQL}
+              tab={
+                <>
+                  <CodeFilled className="mr-2" />
+                  <Text>View SQL</Text>
+                </>
+              }
+            >
+              <BreakdownAnswer {...props} />
+            </Tabs.TabPane>
+            <Tabs.TabPane
+              key="chart"
+              tab={
+                <>
+                  <PieChartFilled className="mr-2" />
+                  <Text>
+                    Chart<Tag className="adm-beta-tag">Beta</Tag>
+                  </Text>
+                </>
+              }
+            >
+              <ChartAnswer {...props} />
+            </Tabs.TabPane>
+          </StyledTabs>
+          <div className="mt-2">
```

Comment:
please assist fix style issue.

```suggestion
          <div className="mt-2 d-flex align-center">
```


![截圖 2025-03-18 下午5 02 25](https://github.com/user-attachments/assets/09a9a2fa-77df-4f11-8135-a6e5afa1f1bd)